src/main/java/org/broadinstitute/hellbender/tools/HaplotypeCallerSpark.java

```diff
@@ -184,11 +184,10 @@ private static void processAssemblyRegions(
 
         final JavaRDD<VariantContext> variants = rdd.mapPartitions(assemblyFunction(header, referenceFileName, hcArgsBroadcast, annotatorEngineBroadcast));
 
-        variants.cache(); // without caching, computations are run twice as a side effect of finding partition boundaries for sorting
```

Comment:
huh, does this have to do with the change in sorting? what sort of difference in runtime does this branch have?

Reply:
The cache call was added to avoid repeat work. I haven't tried re-running benchmarks yet but I suspect it will give some speed up.